build.gradle

```diff
@@ -23,7 +23,7 @@ repositories {
 }
 
 group 'bio.terra'
-version '0.0.37-SNAPSHOT'
+version '0.0.38-SNAPSHOT'
```

Comment:
nit: fwiw, I *think* this is automatically updated by a bot and doesn't need to be touched. but it's also harmless and will just get bumped again afaik.

Reply:
Pretty sure I did not update this myself so maybe the bot already ran?